source/dub/packagesupplier.d

```diff
@@ -23,26 +23,74 @@ import std.file;
 import std.string : format;
 import std.zip;
 
-// TODO: drop the "best package" behavior and let retrievePackage/getPackageDescription take a Version instead of Dependency
+// TODO: Could drop the "best package" behavior and let retrievePackage/
```

Comment:
I was half way into putting that TODO into practice, before I noticed that this would produce some redundant HTTP requests and reverted the changes. I left the comment with a more careful wording, but it really should probably just be removed instead.